examples/resources/bloxone_dhcp_server/resource.tf

```diff
@@ -1,5 +1,40 @@
+
+data "bloxone_dns_hosts" "example_dns_host" {
+  filters = {
+    name = "DNSHost01"
+  }
+}
+
+resource "bloxone_dns_view" "example_dns_view" {
+  name = "example_dns_view"
+}
+
+resource "bloxone_dns_auth_zone" "example_auth_zone" {
+  fqdn         = "domain.com."
+  primary_type = "cloud"
+  internal_secondaries = [
+    {
+      host = data.bloxone_dns_hosts.example_dns_host.results.0.id
+    },
+  ]
+  view = bloxone_dns_view.example_dns_view.id
+}
+
 resource "bloxone_dhcp_server" "example" {
```

Comment:
can you merge this resource with `example_with_options` as well?

Reply:
merged